dao-utils/src/casper_env.rs

```diff
@@ -122,6 +122,6 @@ pub fn get_block_time() -> u64 {
     u64::from(runtime::get_blocktime())
 }
 
-pub fn revert<T: Into<ApiError>>(error: T) {
-    runtime::revert(error)
+pub fn revert<T: Into<ApiError>>(error: T) -> ! {
```

Comment:
Does it change the function's behavior?

Reply:
`runtime::revert` is like panic, it ends the execution. Here I wrapped it with another function and want to have the same behavior, so I need `-> !`.